app/src/main/java/org/oppia/app/walkthrough/WalkthroughActivityListener.kt

```diff
@@ -0,0 +1,7 @@
+package org.oppia.app.walkthrough
+
+/** Listener for when an activity should change pages. */
+interface WalkthroughActivityListener {
+
+  fun previousPage(currentProgress: Int)
```

Comment:
Methods should be verbs or verb phrases, e.g. ``moveToPreviousPage``. Also, both interfaces & their methods should have KDocs per my earlier comment about making it clear how implementations should implement the new functionality. Be sure to also explain circumstances, and document how callers should decide what parameter values to pass in.

Ditto for other new/updated interfaces in this PR.

Reply:
Okay!